_posts/en/podcast/2023-02-16-newsletter-recap.md

```diff
@@ -0,0 +1,1242 @@
+that mess with Bitcoin's incentive mechanism?  I guess the jury's out on that,
+or you think that maybe there's not an issue with that?
+
+**Anthony Towns**: I mean, how is that different to assigning $1 billion to one
+particular UTXO, which we do already by just having however many bitcoin in it?
+
+**Mike Schmidt**: Well, I guess the only difference in that example is that the
+value "in bitcoin" versus the value being in Tether's reserves, or whatnot,
+right, I'm not bringing this up as an actual concern, rather than pointing out
+that there's a difference between those two.
+
+**Anthony Towns**: Yeah, so at some point if the value of bitcoin is trivial and
+miners are getting paid in bitcoin and you just can't get an amount to pay
+people in bitcoin that's not trivial, then maybe that means that the security of
+moving the things that are worth non-trivial amounts on Bitcoin doesn't make
+sense.  But at least to me, it currently feels like that's not as much of a
+concern as it felt like a few weeks ago.  But yeah, that's much more of a
+feeling than something I can argue, I guess.
+
+**Mike Schmidt**: There was another item that was related to block chain data
+storage.  Actually, we have Rodolfo wanting to come in here.
+
+**Anthony Towns**: Just unrelatedly, I've heard a bunch of people in the last
+week or so saying that there's a whole bunch of Ethereum folks firing up nodes
+to play around with ordinals inscriptions and NFTs, and whatnot, and I'm just
+looking at Luke Dashjr's charts of node adoption stuff, and it looks like it's
+gone up from 48,000 nodes a couple of weeks ago to 52,000 nodes, which is a
+relatively big increase, which is interesting.
+
+**Mark Erhardt**: I've also seen a ton of people that seem to have technical
+expertise ask novice questions on Bitcoin Stack Exchange, so they do seem to be
+a bunch of engineers, or people that have at least dabbled with block chain
+stuff, suddenly coming on Stack Exchange to ask questions.
+
+**Mike Schmidt**: Go ahead, Rodolfo.
+
+**Rodolfo Novak**: Hey, guys, I was trying to avoid the topic, but I guess I
+couldn't help myself!  So, I think there's a few things.  One is, my preference
+on this is I don't think it's great to use the block space personally.  There is
+a massive demand for this, and the perception I get from talking to people who
+like this stuff is that the whole point of doing this the way that they're doing
+and talking to Casey as well is that they want to do this the Bitcoin way, which
+is very anchored into the chain, and the inscriptions are done once, and they're
+put in a block and they don't get moved, the sats' ownership moves, and that's
+how they perceive the scarcity value of this, is the fact that it is a very
+early block in their ordinal system thing.  So, that's why they're not
+interested in doing it offchain.
+
+That was essentially why so many of the people doing this craft on Ethereum,
```

Comment:
I suspect that “craft” is not the word that Rodolfo said here, but probably “crap” or “cruft”?

Reply:
crap it is